packages/socket-stream-client/browser.js

```diff
@@ -163,8 +163,9 @@ export class ClientStream extends StreamClientCommon {
     };
 
     const hasSockJS = typeof SockJS === "function";
+    const disableSockJS = __meteor_runtime_config__.DISABLE_SOCKJS;
 
-    this.socket = hasSockJS
+    this.socket = hasSockJS && !disableSockJS
```

Comment:
What happens if `disableSockJS` is true?

Reply:
Then we use native WebSocket instead of SockJS implementation.